### PR TITLE
Fix #4012 : Renaming all colors in colors.xml with correct naming conventions.

### LINKED ISSUE #4012: Rename all colors in colors.xml with correct naming conventions.
Currently ```colors.xml``` contains color definitions which are inconsistent. Make all color declarations consistent by following ```snake_case``` naming convention for all colors.

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

### PR DESCRIPTION
**Pull request by ayush0402**
## Explanation

Fixes #4012 

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+  <color name="concept_toolbar_background">#BE563C</color>
+  <color name="concept_toolbar_heading">#A6503A</color>
   <color name="ic_launcher_background">#26A69A</color>
   <!-- OPPIA COLORS -->
-  <color name="oppiaDarkBlue">#2D4A9D</color>
-  <color name="oppiaPrimaryText">#333333</color>
-  <color name="oppiaPrimaryLight">#009688</color>
-  <color name="oppiaPrimaryText30">#4D333333</color>
-  <color name="oppiaPrimaryTextDark">#DE000000</color>
-  <color name="oppiaPrimaryDark">#00645C</color>
```
Comment (ayush0402) [inline app/src/main/res/values/colors.xml:17]: Removed ```oppiaPrimaryDark``` and renamed all its usage to ```oppia_primary```(previously ```colorPrimary```) as it was re-declaration for same color code.
Same for ```navigationGeneralBackground```.

**Comment by ayush0402** [comment]
@rt4914 @anandwana001 PTAL.

**Comment by anandwana001** [comment]
I will be able to take a look at this on weekend.

**Review by anandwana001 — APPROVED** [review]
my codeowners files, LGTM

**Comment by BenHenning** [comment]
color_defs.xml LGTM for snake_case changes & replacing colors.xml.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**